.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+local.properties
```

Comment:
This .gitignore file is the file that was automatically generated for my own project, this is needed as files such as local.properties are a PER MACHINE file, and trying to run the project on my machine resulted in an error as it was trying to point to the file location of the android sdk on your device.

Reply:
Good discussion.  Good idea.